developer-tools.md

```diff
@@ -397,6 +397,13 @@ Other tips:
 - "Rebuild Project" can fail the first time the project is compiled, because generate source files 
 are not automatically generated. Try clicking the "Generate Sources and Update Folders For All 
 Projects" button in the "Maven Projects" tool window to manually generate these sources.
+- The version of Maven bundled with IntelliJ may not be new enough for Spark. If that happens,
+the action "Generate Sources and Update Folders For All Projects" could fail silently. 
+Please remember to reset the Maven home directory 
+(`Preference -> Build, Execution, Deployment -> Maven -> Maven home directory`) of your project to 
+point to a newer installation of Maven. You may also build Spark with script `build/mvn` first.
```

Comment:
with script -> with the script
install a proper Maven -> install a recent version of Maven

Reply:
Thanks. I updated the developer-tools.md accordingly, but not yet run the build to update corresponding html yet. Will do that tonight when I can access a desktop.

Reply:
site/developer-tools.html is also updated. please kindly take a look.